site/styles/_variables.scss

```diff
@@ -56,12 +56,12 @@ $navbar-dark-color:         $body-color !default;
 $navbar-dark-hover-color:   #FD4766 !default;
 
 // Navbar Logo (Brand)
-$navbar-brand-padding-y: 15px !default;
+$navbar-brand-padding-y: 25px !default;
```

Comment:
Since this increases the nav height by 20px (10 above and 10 below), you should probably add 20px to these too:

`$carousel-height`
`$carousel-height-sm`
`$carousel-height-md`
`$carousel-caption-top`
`$carousel-caption-top-md`
`$page-padding-top`
`$page-padding-top-md`
`$page-detail-padding-top`
`$page-detail-padding-top-md`